debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlConnectorRegressionIT.java

```diff
@@ -811,6 +811,83 @@ public void shouldConsumeAllEventsFromDecimalTableInDatabaseUsingBinlogAndNoSnap
         });
     }
 
+    @Test
+    @FixFor("DBZ-611")
+    public void shouldConsumeDecimalAsStringFromBinlog() throws SQLException, InterruptedException {
+        // Use the DB configuration to define the connector's configuration ...
+        config = DATABASE.defaultConfig()
+                              .with(MySqlConnectorConfig.TABLE_WHITELIST, DATABASE.qualifiedTableName("dbz_147_decimalvalues"))
+                              .with(MySqlConnectorConfig.INCLUDE_SCHEMA_CHANGES, true)
+                              .with(MySqlConnectorConfig.SNAPSHOT_MODE, SnapshotMode.NEVER.toString())
+                              .with(MySqlConnectorConfig.DECIMAL_HANDLING_MODE, DecimalHandlingMode.STRING)
+                              .build();
+        // Start the connector ...
+        start(MySqlConnector.class, config);
+
+        // ---------------------------------------------------------------------------------------------------------------
+        // Consume all of the events due to startup and initialization of the database
+        // ---------------------------------------------------------------------------------------------------------------
+        // Testing.Debug.enable();
+        int numCreateDatabase = 1;
+        int numCreateTables = 9; // still read DDL for all tables
+        int numDataRecords = 1;
```

Comment:
Where's that data record coming from? I'd somehow have expected to see and INSERT + UPDATE + DELETE statement here.

Reply:
https://github.com/debezium/debezium/blob/master/debezium-connector-mysql/src/test/resources/ddl/regression_test.sql#L113-L120

Reply:
Ah yes, of course :)